config/article_scrape_patterns.yml

```diff
@@ -11,6 +12,28 @@ data_types:
 # All data types must include :method and :pattern
 #
 presets:
+  author:
+    class: &class_author
+      method: xpath
+      pattern: "//*[contains(concat(' ', @class, ' '), 'author')]"
```

Comment:
might be easier to read like this:
```
contains(                        # checks to see if class has author in it
  concat(' ', @class, ' '), # will concat blank spaces on either end
  'author'
)
```

I was basing this off of http://stackoverflow.com/questions/1390568/how-can-i-match-on-an-attribute-that-contains-a-certain-string
but we can use xpath 2.0 http://stackoverflow.com/questions/1390568/how-can-i-match-on-an-attribute-that-contains-a-certain-string/12165195#12165195

will switch